### ISSUE TITLE
Consufing error message with the `@initialization` macro

### ISSUE THREAD

**Issue by bvdmitri** [ReactiveBayes/RxInfer.jl#279]
Not sure if its easy to fix, but this 
```julia
@initialization begin 
    q(μ) = NormalMeanPrecision(0.0, 0.001),
    q(τ) = GammaShapeRate(10.0, 10.0)
end
```
gives this error
```
ERROR: syntax: "0" is not a valid function argument name around REPL[4]:2
Stacktrace:
 [1] top-level scope
   @ REPL[4]:1
```

**Comment by wouterwln** [comment]
Drop the `,` after the marginal for mu. Do you want me to investigate to improve the error msg?

**Comment by wouterwln** [comment]
Its fixed in dev-3.0.0 btw

**Comment by bvdmitri** [comment]
Ah, sorry, didn't check! Great, thanks!!
